nbextensions/usability/hinterland/hinterland.js

```diff
@@ -69,8 +70,13 @@ define(function (require, exports, module) {
 						}, cur);
 
 						if (pre_cursor !== '' && !ignore_re.test(pre_cursor)) {
-							completer.startCompletion();
-							completer.autopick = false;
+							if (pre_cursor === '(') {
+								cell.tooltip.request(cell);
```

Comment:
[This is a comment without trying the new functionality...]
The right hints are in the completer, they are just buried under the global stuff. So if it would be possible to simple sort the completion tipps so that `xxxx=` (equal sign == parameter) are at the top, this would be even better (because you could then select the param)... IMO this could even be done in all cases: if the cursor is not in the context of a function call, then there will simple no such items...